app/controllers/users_controller.rb

```diff
@@ -143,13 +143,24 @@ def remove_identity
   def onboarding_update
     if params[:user]
       sanitize_user_params
-      permitted_params = %i[summary location employment_title employer_name last_onboarding_page]
-      current_user.assign_attributes(params[:user].permit(permitted_params))
+      permitted_user_params = %i[last_onboarding_page]
```

Comment:
**suggestion (non-blocking):** As always, I'd feel slightly better if we moved this out of the method and into a constant.

Reply:
✅